src/Extensions/AzureBlobStorage/AzureBlobStorage.csproj

```diff
@@ -19,6 +19,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\..\Microsoft.Diagnostics.Monitoring.Options\Microsoft.Diagnostics.Monitoring.Options.csproj" />
```

Comment:
Please move the display strings from Microsoft.Diagnostics.Monitoring.Options to each of the individual extensions.